acl_loader/main.py

```diff
@@ -583,6 +586,41 @@ def convert_input_interface(self, table_name, rule_idx, rule):
 
         return rule_props
 
+    def is_db_rule_dynamic(self, key):
+        try:
+            data = self.configdb.get_entry(self.ACL_RULE, key)
+            if 'is_time_based' in data and data['is_time_based'].lower() == 'true':
+                return True
+            return False
+        except Exception as e:
+            return False
+        
+    def is_time_based_rule(self, rule):
+        return rule.time_range.config.start_time
+
+    def convert_dynamic_acl_config(self, key, rule):
+        rule_props = {}
+        
+        if self.is_time_based_rule(rule):
+            rule_props['is_time_based'] = 'True'
+            rule_props['start_time'] = rule.time_range.config.start_time
+            rule_props['end_time'] = rule.time_range.config.end_time
+        
+        return rule_props
+
+    def add_config_db_dynamic_acl_rule_entry(self, table, rule, entry):
+        """
+        Insert entry to CONFIG_DB for dynamic ACL
+        """
+        key = table + '|' + rule
+        try:
+            entry.pop('is_time_based', None)
```

Comment:
Why we have this entry? It is not exist in HLD

Reply:
"is_time_based" is an internal field to mark if the ACL is time-based not or during acl-loader parsing the json file.